lib/app/views/dashboard.erb

```diff
@@ -15,13 +17,28 @@
           <% if current_user.ongoing.any? %>
             <p>Keep working on these problems, or mark them as done.</p>
             <ul class="nav nav-stacked" id="menu-current" role="menu">
-              <% current_user.ongoing.each do |submission| %>
+              <% current_user.ongoing[0..4].each do |submission| %>
```

Comment:
I think this is going to request ALL of the exercises, and then just display the first 5. Let's use limit so that we don't actually pull back data from the database that we're not going to show